### DISCUSSION TITLE
In Firefox's default horizontal tab bar, always open a tab at the end (right hand side)

### DISCUSSION THREAD

**Discussion by Waterford-Wanderer** [mbnuqw/sidebery#1857] [Q&A]
I apologise in advance for being ignorant and stupid, but is there an easy or quick way to retain the behaviour I had already set in the "**about:config**" of Firefox, where any new or child tab always opens **at the end** (right hand side) of Firefox's default horizontal tab bar (when Sidebery is active) ?

**Comment by albino1** [comment]
Take a look at `Settings` -> `Tabs` -> `Position of new tab`

**Comment by Waterford-Wanderer** [comment]
Thank you !

[ I changed Settings -> Tabs -> Position of new tab  -> Place New Tab (general rule) to "panel end"
and that seems to have restored things so that any new or child tab always opens at the end (right hand side) of Firefox's default horizontal tab bar (when Sidebery is active) ]
